.github/workflows/ci.yml

```
@@ -62,3 +62,60 @@ jobs:
       - name: Test
         run: |
           cmake --build build --target check --config "${{ matrix.cfg.type }}" -j 2
+
+
+  linux:
+    runs-on: ubuntu-latest
+    name: ${{ matrix.platform.str }}-${{ matrix.cfg.str }}
+    strategy:
+      matrix:
+        platform:
+          - { str: linux-x64 }
+        cfg:
+          - { external: OFF, type: RelWithDebInfo, str: internal-release }
+          - { external: OFF, type: Debug, str: internal-debug }
+          - { external: ON, type: RelWithDebInfo, str: external-release }
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          submodules: true
+
+      - name: Install dependencies
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y \
+              autoconf \
+              cmake \
+              libsecret-1-dev \
+              libtool \
+              nasm \
+              ninja-build \
+              qtbase5-dev
+
+      - name: Configure
+        run: |
+          cmake \
+            -DUSE_VCPKG=ON \
```

```suggestion
            -GNinja \
            -DUSE_VCPKG=ON \
```

Since we install it, we may as well use it :smile: